src/api/overTheBox/overTheBox.v6.service.js

```diff
@@ -109,6 +109,20 @@ angular.module('ovh-api-services').service('OvhApiOverTheBoxV6', ($resource, Ovh
       url: '/overTheBox/:serviceName/device/logs ',
       isArray: false,
     },
+    getActions: {
```

Comment:
You should delete these ones as they at in the `src/api/overTheBox/device/overTheBox-device.v6.service.js` file 🙂